frodo.js

```diff
@@ -7,7 +7,6 @@
  * See README.md for details.
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?